spec/models/miq_server/worker_management/heartbeat_spec.rb

```diff
@@ -15,5 +15,75 @@
         expect(worker.reload.last_heartbeat).to be_within(1.second).of(t)
       end
     end
+
+    context "when using file based heartbeating" do
+      let!(:first_heartbeat) { Time.now.utc }
+
+      around do |example|
+        ENV["WORKER_HEARTBEAT_METHOD"] = "file"
+        ENV["WORKER_HEARTBEAT_FILE"]   = "/path/to/worker.hb"
```

Comment:
@jrafanie I thought `lets` weren't able to be accessed within an "around" block, but I could be wrong.  But that was my rational for it not being DRY.